### ISSUE TITLE
Finish porting .eu content - fix auto-imported posts

### ISSUE THREAD

**Issue by NickSto** [galaxyproject/galaxy-hub#1708]
With #1707, all news and events posts from .eu are now present on .org. But since we didn't go through each of those to make sure they display properly, there will be a lot of display issues. There's still a lot of work to do to get them to work properly. Some of it could be automated but I don't think all of it can be.

## Missing features

.eu used the Liquid template engine, which allowed inserting elements inline in the Markdown files. When you see something like this on a page, that's what it is:
{% include map.html location=page.location showmap=true zoomlevel=15 hidepopup=true %}

These can usually be re-implemented as Vue components. Some, like `{% twitter %}`, already have been (see [src/components/Twitter.vue](https://github.com/galaxyproject/galaxy-hub/blob/master/src/components/Twitter.vue)). #1138 lists all the features I've discovered.

## Broken syntax

Because the .eu site's Markdown used Kramdown syntax and we use CommonMark, there are some incompatibilities.

### HTML/Markdown mixing

Some Markdown may not display because there's HTML in an adjacent line. See [here](https://galaxyproject.org/hub/authoring/#inserting-html) for details on how you need to separate them (with a blank line).

### Tables

One common issue is that tables don't display. That's usually because the header omits some vertical pipes. That can be easily fixed, though. For example, you might find this table in the Markdown:

```markdown
example | header | fields
------- | ------ | ------
cell    | value  | examples
```
The minimum you need to do is to add a `|` on either end of the header separator:
```markdown
example | header | fields
| ----- | ------ | ------ |
cell    | value  | examples
```

### Inline Attribute Lists

Kramdown has a feature called [Inline Attribute Lists](https://kramdown.gettalong.org/syntax.html#block-ials). This lets you add HTML attributes to parts of your Markdown. Luckily, we support a version of this with the [remark-attr](https://www.npmjs.com/package/remark-attr) plugin. Here's an example:
```Markdown
[link text](/path/){:.fa .fa-slideshare}
```
This adds the classes `fa` and `fa-slideshare` to the `<a>` element.

Unfortunately, there are some minor differences. The main one seems to be that Kramdown doesn't require a space between the opening `{:` and the attributes. What happens is that the first attribute gets ignored and only once it sees a space does it start actually using what you specify. This is easily fixed, of course, and could be automated.

**Comment by itisAliRH** [comment]
Closing this as superseded by the completed `.eu` migration.

The auto-imported posts were re-processed during the 2026 migration: Liquid-era constructs were converted to the Hub's MDX components, images were re-pathed to Hub media locations, and the ~200 re-dated events and ~90 rewritten slugs are all covered by exact redirect rules so the old URLs still resolve (verified in prod).

Every record is accounted for in the migration audit manifest with zero unclassified entries. If a specific old post still renders badly, a targeted issue with the URL is much more actionable than this umbrella one; please open one, and I'll fix it.
